pladder/bot.py

```diff
@@ -45,6 +45,24 @@ def new_command_group(self, name):
         return self.commands.new_command_group(name)
 
     def RunCommand(self, timestamp, network, channel, nick, text):
+        prefilter_result = self.RunCommand_internal(timestamp, network, channel, nick, text)
+        if prefilter_result['command'] != "error" and self.commands.lookup_command("output-filter"):
```

Comment:
The `"command"` entry in the dictionary is a piece of dead code once used by the logging system (so that `search-log` wouldn't appear in the log). It always had to have a value, so we put `"error"` in there if the execution didn't complete to fill the gap with something. This entry will likely go away in the future.

The proper way to detect an error is to check if `interpret()` raises an exception or not.

Reply:
I removed the "display_name" result from `interpret()` so that (hopefully) no one else gets confused by it in the future. (It's still present in the DBus API, though. Changing it is a breaking change that needs some manual intervention.)